File: vcc-sdk-java/src/main/java/com/ctrip/ccard/creditcard/vcc/bean/SettlementInfo.java
package com.ctrip.ccard.creditcard.vcc.bean;

/**
 * Description:
 */
public class SettlementInfo {
    /**
     * 交易发生时间
     */
    private String occurTime;

    /**
     * 入账日期
     */
    private String postDate;

    /**
     * 出账日期
     */
    private String settleDate;

    /**
     * 交易币种
     */
    private String transCurrency;

    /**
     * 交易金额
     */
    private String transAmt;

    /**
     * 卡币种
     */
    private String localCurrency;

    /**
     * 卡本币种交易金额
     */
    private String localCurrencyAmt;

    /**
     * 入账币种
     */
    private String billCurrency;

    /**
     * 入账币种对应金额
     */
    private String billCurrencyAmt;

    /**
     * 入账原金额
     */
    private String originalBillCurrencyAmt;

    /**
     * 交易类型
     */
    private String transType;

    /**
     * 交易类型码
     */
    private String transCode;

    /**
     * 授权码
     */
    private String authNo;

    /**
     * 交易流水号
     */
    private String transNo;

    /**
     * 是否是收或付标识
     */
    private String isCredit;

    /**
     * 扣款商户信息
     */
    private TransMerchantInfo transMerchantInfo;

    /**
     * 用于今后的扩展
     */
    private String transExtra;

    public String getOccurTime() {
        return occurTime;
    }

    public void setOccurTime(String occurTime) {
        this.occurTime = occurTime;
    }

    public String getPostDate() {
        return postDate;
    }

    public void setPostDate(String postDate) {
        this.postDate = postDate;
    }

    public String getSettleDate() {
        return settleDate;
    }

    public void setSettleDate(String settleDate) {
        this.settleDate = settleDate;
    }

    public String getTransCurrency() {
        return transCurrency;
    }

    public void setTransCurrency(String transCurrency) {
        this.transCurrency = transCurrency;
    }

    public String getTransAmt() {
        return transAmt;
    }

    public void setTransAmt(String transAmt) {
        this.transAmt = transAmt;
    }

    public String getLocalCurrency() {
        return localCurrency;
    }

    public void setLocalCurrency(String localCurrency) {
        this.localCurrency = localCurrency;
    }

    public String getLocalCurrencyAmt() {
        return localCurrencyAmt;
    }

    public void setLocalCurrencyAmt(String localCurrencyAmt) {
        this.localCurrencyAmt = localCurrencyAmt;
    }

    public String getBillCurrency() {
        return billCurrency;
    }

    public void setBillCurrency(String billCurrency) {
        this.billCurrency = billCurrency;
    }

    public String getBillCurrencyAmt() {
        return billCurrencyAmt;
    }

    public void setBillCurrencyAmt(String billCurrencyAmt) {
        this.billCurrencyAmt = billCurrencyAmt;
    }

    public String getOriginalBillCurrencyAmt() {
        return originalBillCurrencyAmt;
    }

    public void setOriginalBillCurrencyAmt(String originalBillCurrencyAmt) {
        this.originalBillCurrencyAmt = originalBillCurrencyAmt;
    }

    public String getTransType() {
        return transType;
    }

    public void setTransType(String transType) {
        this.transType = transType;
    }

    public String getTransCode() {
        return transCode;
    }

    public void setTransCode(String transCode) {
        this.transCode = transCode;
    }

    public String getAuthNo() {
        return authNo;
    }

    public void setAuthNo(String authNo) {
        this.authNo = authNo;
    }

    public String getTransNo() {
        return transNo;
    }

    public void setTransNo(String transNo) {
        this.transNo = transNo;
    }

    public String getIsCredit() {
        return isCredit;
    }

    public void setIsCredit(String isCredit) {
        this.isCredit = isCredit;
    }

    public TransMerchantInfo getTransMerchantInfo() {
        return transMerchantInfo;
    }

    public void setTransMerchantInfo(TransMerchantInfo transMerchantInfo) {
        this.transMerchantInfo = transMerchantInfo;
    }

    public String getTransExtra() {
        return transExtra;
    }

    public void setTransExtra(String transExtra) {
        this.transExtra = transExtra;
    }
}

File: php-sdk/src/triplink/vcc/card/client/query/SettlementInfo.php
<?php
/**
 * Created by PhpStorm.
 * User: wang.xiaoliang
 * Date: 2021/8/5
 * Time: 13:58
 */
namespace triplink\vcc\card\client\query;

class SettlementInfo
{
    /**
     * 交易发生时间
     */
    var $occurTime;

    /**
     * 入账日期
     */
    var $postDate;

    /**
     * 出账日期
     */
    var $settleDate;

    /**
     * 交易币种
     */
    var $transCurrency;

    /**
     * 交易金额
     */
    var $transAmt;

    /**
     * 卡币种
     */
    var $localCurrency;

    /**
     * 卡本币种交易金额
     */
    var $localCurrencyAmt;

    /**
     * 入账币种
     */
    var $billCurrency;

    /**
     * 入账币种对应金额
     */
    var $billCurrencyAmt;

    /**
     * 入账原金额
     */
    var $originalBillCurrencyAmt;

    /**
     * 交易类型
     */
    var $transType;

    /**
     * 交易类型码
     */
    var $transCode;

    /**
     * 授权码
     */
    var $authNo;

    /**
     * 交易流水号
     */
    var $transNo;

    /**
     * 是否是收或付标识
     */
    var $isCredit;

    /**
     * 扣款商户信息
     */
    var $transMerchantInfo;

    /**
     * 用于今后的扩展
     */
    var $transExtra;

    /**
     * @return mixed
     */
    public function getOccurTime()
    {
        return $this->occurTime;
    }

    /**
     * @param mixed $occurTime
     */
    public function setOccurTime($occurTime): void
    {
        $this->occurTime = $occurTime;
    }

    /**
     * @return mixed
     */
    public function getPostDate()
    {
        return $this->postDate;
    }

    /**
     * @param mixed $postDate
     */
    public function setPostDate($postDate): void
    {
        $this->postDate = $postDate;
    }

    /**
     * @return mixed
     */
    public function getSettleDate()
    {
        return $this->settleDate;
    }

    /**
     * @param mixed $settleDate
     */
    public function setSettleDate($settleDate): void
    {
        $this->settleDate = $settleDate;
    }

    /**
     * @return mixed
     */
    public function getTransCurrency()
    {
        return $this->transCurrency;
    }

    /**
     * @param mixed $transCurrency
     */
    public function setTransCurrency($transCurrency): void
    {
        $this->transCurrency = $transCurrency;
    }

    /**
     * @return mixed
     */
    public function getTransAmt()
    {
        return $this->transAmt;
    }

    /**
     * @param mixed $transAmt
     */
    public function setTransAmt($transAmt): void
    {
        $this->transAmt = $transAmt;
    }

    /**
     * @return mixed
     */
    public function getLocalCurrency()
    {
        return $this->localCurrency;
    }

    /**
     * @param mixed $localCurrency
     */
    public function setLocalCurrency($localCurrency): void
    {
        $this->localCurrency = $localCurrency;
    }

    /**
     * @return mixed
     */
    public function getLocalCurrencyAmt()
    {
        return $this->localCurrencyAmt;
    }

    /**
     * @param mixed $localCurrencyAmt
     */
    public function setLocalCurrencyAmt($localCurrencyAmt): void
    {
        $this->localCurrencyAmt = $localCurrencyAmt;
    }

    /**
     * @return mixed
     */
    public function getBillCurrency()
    {
        return $this->billCurrency;
    }

    /**
     * @param mixed $billCurrency
     */
    public function setBillCurrency($billCurrency): void
    {
        $this->billCurrency = $billCurrency;
    }

    /**
     * @return mixed
     */
    public function getBillCurrencyAmt()
    {
        return $this->billCurrencyAmt;
    }

    /**
     * @param mixed $billCurrencyAmt
     */
    public function setBillCurrencyAmt($billCurrencyAmt): void
    {
        $this->billCurrencyAmt = $billCurrencyAmt;
    }

    /**
     * @return mixed
     */
    public function getOriginalBillCurrencyAmt()
    {
        return $this->originalBillCurrencyAmt;
    }

    /**
     * @param mixed $originalBillCurrencyAmt
     */
    public function setOriginalBillCurrencyAmt($originalBillCurrencyAmt): void
    {
        $this->originalBillCurrencyAmt = $originalBillCurrencyAmt;
    }

    /**
     * @return mixed
     */
    public function getTransType()
    {
        return $this->transType;
    }

    /**
     * @param mixed $transType
     */
    public function setTransType($transType): void
    {
        $this->transType = $transType;
    }

    /**
     * @return mixed
     */
    public function getTransCode()
    {
        return $this->transCode;
    }

    /**
     * @param mixed $transCode
     */
    public function setTransCode($transCode): void
    {
        $this->transCode = $transCode;
    }

    /**
     * @return mixed
     */
    public function getAuthNo()
    {
        return $this->authNo;
    }

    /**
     * @param mixed $authNo
     */
    public function setAuthNo($authNo): void
    {
        $this->authNo = $authNo;
    }

    /**
     * @return mixed
     */
    public function getTransNo()
    {
        return $this->transNo;
    }

    /**
     * @param mixed $transNo
     */
    public function setTransNo($transNo): void
    {
        $this->transNo = $transNo;
    }

    /**
     * @return mixed
     */
    public function getisCredit()
    {
        return $this->isCredit;
    }

    /**
     * @param mixed $isCredit
     */
    public function setIsCredit($isCredit): void
    {
        $this->isCredit = $isCredit;
    }

    /**
     * @return mixed
     */
    public function getTransMerchantInfo()
    {
        return $this->transMerchantInfo;
    }

    /**
     * @param mixed $transMerchantInfo
     */
    public function setTransMerchantInfo($transMerchantInfo): void
    {
        $this->transMerchantInfo = $transMerchantInfo;
    }

    /**
     * @return mixed
     */
    public function getTransExtra()
    {
        return $this->transExtra;
    }

    /**
     * @param mixed $transExtra
     */
    public function setTransExtra($transExtra): void
    {
        $this->transExtra = $transExtra;
    }


}

?>